demo/run.sh

```diff
@@ -19,35 +19,34 @@ function main() {
   createDemoEnvironment
   waitForServer
   createHostFactoryToken
-  runAnsible
-  retrieveSecretInTarget
+  conjurizeTargetContainer
+  RetrieveSecretInTargetWithCli
+  RetrieveSecretInTargetWithSummon
+  RetrieveSecretWithPlugin
 }
 
 function createDemoEnvironment() {
   echo "Creating demo environment"
   echo '-----'
 
-  docker-compose build --pull target
+  docker-compose build --pull target lookup-target
   docker-compose pull postgres conjur client
-  docker-compose up -d client postgres conjur target
-
-  echo '-----'
+  docker-compose up -d client postgres conjur target lookup-target
 }
 
 function waitForServer() {
-  echo 'Waiting for conjur server to be healthy'
   echo '-----'
+  echo 'Waiting for conjur server to be healthy'
```

Comment:
Do we not have a `HEALTHCHECK` in `cyberark/conjur` image yet?

Reply:
HEALTHCHECK in not what we want/need. We need to continue using a wait script until Docker makes a change to Docker Compose startup.

Reply:
Yes.  It's not coming anytime soon, because HEALTHCHECK isn't what we need here.  HEALTHCHECK will remove any unhealthy containers from a docker-compose cluster.  What we really need is a READY entry point.